.travis.yml

```diff
@@ -3,17 +3,17 @@ sudo: required
 env:
     matrix:
         # openssl tests
-        - COMP=openssl OS_TYPE=centos OS_VERSION=5
+        #- COMP=openssl OS_TYPE=centos OS_VERSION=5
         - COMP=openssl OS_TYPE=centos OS_VERSION=6
         - COMP=openssl OS_TYPE=centos OS_VERSION=7
         - COMP=openssl OS_TYPE=fedora OS_VERSION=24
```

Comment:
Regarding the recent release of Fedora 25: do you prefer explicit version assignment (24, 25, etc.) or the `latest` tag would be sufficient? With the `latest` tag we would avoid bumping the version every ~6 months ([source](https://hub.docker.com/r/library/fedora/)).